tests/Middleware/AuthTokenMiddlewareTest.php

```diff
@@ -282,6 +259,8 @@ public function testShouldNotifyTokenCallback(callable $tokenCallback)
         $this->mockFetcher->fetchAuthToken(Argument::any())
             ->shouldBeCalledTimes(1)
             ->willReturn($cachedValue);
+        $this->mockFetcher->getLastReceivedToken()
+            ->willReturn($cachedValue);
```

Comment:
This additonal prophecy is now necessary as we are introducting another check for `$token` from `getLastReceivedToken` before calling `tokenCallback`.